lib/src/app.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../src/blocs/login_blocs/login_blocs.dart';
+import '../src/blocs/login_blocs/login_event.dart';
+import '../src/blocs/authentication_blocs/authentication_blocs.dart';
+import '../src/blocs/authentication_blocs/authentication_event.dart';
+import '../src/blocs/authentication_blocs/authentication_state.dart';
+import '../src/blocs/bloc_observer.dart';
+import '../src/repositories/user_repository.dart';
+import '../src/screens/home_screen.dart';
+
+import 'screens/onboarding_screen.dart';
+import 'screens/login_screen.dart';
+import 'screens/forgot_password_screen.dart';
+import 'screens/signup_screen.dart';
+
+class App extends StatelessWidget {
+  final UserRepository userRepository = UserRepository();
```

Comment:
what is this?

Reply:
When i rebuild bloc and it miss to erased this line